src/Frontend/src/components/configuration/PlatformConnections.vue

```diff
@@ -94,9 +94,9 @@ function saveConnections() {
                 <div class="col-7 form-group">
                   <label for="monitoringUrl"
                     >CONNECTION URL
-                    <span class="auxilliary-label">(OPTIONAL) ( Enter ! to disable monitoring)</span>
+                    <span class="auxilliary-label">(OPTIONAL) (Enter ! to disable monitoring)</span>
                     <template v-if="monitoringConnectionState.unableToConnect && !useIsMonitoringDisabled()">
-                      <span class="failed-validation"> <i class="fa fa-exclamation-triangle"></i> Unable to connect </span>
+                      &nbsp;<span class="failed-validation"> <i class="fa fa-exclamation-triangle"></i> Unable to connect </span>
```

Comment:
this shouldn't be here? Spacing should be controlled with styles

Reply:
fixed with styles